library/core/src/main/java/com/google/android/exoplayer2/audio/MediaCodecAudioRenderer.java

```diff
@@ -737,6 +768,9 @@ public void handleMessage(@MessageType int messageType, @Nullable Object message
       case MSG_SET_VOLUME:
         audioSink.setVolume((Float) message);
         break;
+      case MSG_SET_CODEC_PARAMETERS:
```

Comment:
This probably needs handling in the base class (`MediaCodecRenderer`). Note also that the decoder is recreated in some cases, and also may not exist when this message is received. You'll need to correctly handle these cases. Probably by configuring the decoder correctly somewhere under `initCodec`.

Reply:
Or call setDialogEnhancement() from the MediaCodecAudioRenderer.onCodecInitialized() callback

Eventually the setDialogEnhancement could be called from a method that applies any "managed" params and that method could be called in updateMediaCodecParameters() as well as in onCodecInitialized()